DOCS-620: Reorganize ML content (#938)
viamrobotics/docs: docs/manage/data/view.md

To export your captured data from the cloud, see [Export Data](../export).

> If you have collected image data, you can use that data to [train image classification model](../../ml/train-model) within Viam.

Reviewer: ```suggestion
Cloud image data can be used to [train machine learning models](../../ml/train-model) within Viam.
```
Since this is really about data that was synced to the cloud

Author: ```suggestion
You can use cloud image data to [train machine learning models](../../ml/train-model) within Viam.
```
